Dockerfile build step fails due to misaligned Go versions (1.21.5 vs 1.22.0)

<img src="https://raw.githubusercontent.com/derailed/k9s/master/assets/k9s_err.png" align="right" width="100" height="auto"/>

<br/>
<br/>
<br/>

**Describe the bug**
The Docker build step is currently failing because the image's Go version is misaligned with the mod file:
```
3.948 go: go.mod requires go >= 1.22.0 (running go 1.21.5; GOTOOLCHAIN=local)
3.949 make: *** [Makefile:28: build] Error 1
```

Appears to have been introduced [in v0.32.5](https://github.com/derailed/k9s/commit/3f30a706887ccd42e85344824132e3d3a9354b97#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6L3).

**To Reproduce**
Steps to reproduce the behavior:

1. Checkout master
2. `docker build -t k9s-docker:v0.0.1 .` (as per documentation)

**Expected behavior**
Docker image should be built successfully.

**Versions (please complete the following information):**

- OS: [Pop OS 22.04]
- K9s: [ v0.32.5]
- K8s: [v1.30.0]

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,7 +1,7 @@
 # -----------------------------------------------------------------------------
 # The base image for building the k9s binary
 
-FROM golang:1.21.5-alpine3.17 AS build
+FROM golang:1.22-alpine3.20 AS build
 
 WORKDIR /k9s
 COPY go.mod go.sum main.go Makefile ./

